Documentation/reports.md

```diff
@@ -241,7 +241,33 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ```
 
+### expiration 
+
+Add the expiration field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the expiration duration value. The retention period is equal to the Report creationDate plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report. Deleting the Report from the cluster can take several minutes.
+	
+Setting the expiration field is not recommended for roll-up or aggregated reports. If a Report is depended upon by other Reports or ReportQueries, then the Report is not removed at the end of the retention period. You can view the reporting-operator logs at debug level for the timing output around a Report retention decision.
+
+For example, the following scheduled Report is deleted 30 minutes after the creationDate of the Report:
+```yaml
```

Comment:
Add a newline between the code block and the above description of the code block.